tests/basic_tests.cpp

```diff
@@ -430,4 +430,14 @@ TYPED_TEST(basic_tests, nodejs_undici_2971) {
       out->get_href(),
       R"(https://non-ascii-location-header.sys.workers.dev/%EC%95%88%EB%85%95)");
   SUCCEED();
+}
+
+TYPED_TEST(basic_tests, path_setter_bug) {
+  std::string_view base = "blob:/?";
+  auto base_url = ada::parse<ada::url_aggregator>(base);
```

Comment:
```suggestion
  auto base_url = ada::parse<TypeParam>(base);
```

Reply:
Can't do that as is. It will break the build.

Reply:
Ah ok. Then maybe we shouldn't use typed_test